librz/analysis/p/analysis_x86_cs.c

```diff
@@ -3118,45 +3123,53 @@ static int cs_len_prefix_opcode(uint8_t *item) {
 }
 
 static int analop(RzAnalysis *a, RzAnalysisOp *op, ut64 addr, const ut8 *buf, int len, RzAnalysisOpMask mask) {
-	static int omode = 0;
-#if USE_ITERZ_API
-	static
-#endif
-		cs_insn *insn = NULL;
-	int mode = (a->bits == 64) ? CS_MODE_64 : (a->bits == 32) ? CS_MODE_32
-		: (a->bits == 16)                                 ? CS_MODE_16
-								  : 0;
+	X86CSContext *ctx = (X86CSContext *)a->plugin_data;
+
+	int mode = 0;
+	switch (a->bits) {
+	case 64:
+		mode = CS_MODE_64;
+		break;
+	case 32:
+		mode = CS_MODE_32;
+		break;
+	case 16:
+		mode = CS_MODE_16;
+		break;
+	default:
+		break;
+	}
 	int n, ret;
 
-	if (handle && mode != omode) {
-		if (handle != 0) {
-			cs_close(&handle);
-			handle = 0;
+	if (ctx->handle && mode != ctx->omode) {
+		if (ctx->handle != 0) {
+			cs_close(&ctx->handle);
+			ctx->handle = 0;
 		}
 	}
-	omode = mode;
-	if (handle == 0) {
-		ret = cs_open(CS_ARCH_X86, mode, &handle);
+	ctx->omode = mode;
+	if (ctx->handle == 0) {
+		ret = cs_open(CS_ARCH_X86, mode, &ctx->handle);
 		if (ret != CS_ERR_OK) {
-			handle = 0;
+			ctx->handle = 0;
 			return 0;
 		}
 	}
 	op->cycles = 1; // aprox
-	cs_option(handle, CS_OPT_DETAIL, CS_OPT_ON);
+	cs_option(ctx->handle, CS_OPT_DETAIL, CS_OPT_ON);
 	// capstone-next
 #if USE_ITERZ_API
```

Comment:
Is this even used `USE_ITERZ_API`? It looks like zero everywhere, just remove this code.